application/AccessAppService.go

```diff
@@ -17,11 +17,11 @@ type AccessAppService struct {
 
 // CheckRequest is an actual request to check for permissions.
 type CheckRequest struct {
-	Requestor    string `validate:"required,spicedb"`
-	Subject      string `validate:"required,spicedb"`
-	ResourceType string `validate:"required,spicedb"`
-	ResourceID   string `validate:"required,spicedb"`
-	Operation    string `validate:"required"` //How to constrain further? List of values?
+	Requestor    string `validate:"required,spicedb-id"`
+	Subject      string `validate:"required,spicedb-id"`
+	ResourceType string `validate:"required,spicedb-type"`
+	ResourceID   string `validate:"required,spicedb-id"`
+	Operation    string `validate:"required,spicedb-permission"` //How to constrain further? List of values?
```

Comment:
a thought: we could further narrow this down here. basically the check call should only allow the "access" operation for this specific use case, right? So we could basically deny everything else as bad request.